peer/brontide.go

```diff
@@ -1258,15 +1258,13 @@ func (p *Brontide) Disconnect(reason error) {
 
 	p.log.Infof(err.Error())
 
+	// Stop ping manager before closing TCP connection
```

Comment:
nit: missing full stop

Reply:
Fwiw, as far as I can tell, the full-stop rule is only important for godocs since it affects the godoc rendering semantics. In-line comments like this aren't rendered and so I don't think it matters, but others may know things that I don't.

CC @ellemouton @yyforyongyu